caddy/main.go

```diff
@@ -20,6 +20,8 @@
 package main
 
 import "github.com/mholt/caddy/caddy/caddymain"
+import _ "github.com/krader1961/caddy-geoip"
+import _ "github.com/krader1961/ipfilter"
```

Comment:
This seems wrong

Reply:
/facepalm /embarrassment Yes, my custom config commit should not have been included. I've force pushed the proper change.